deployments/scripts/push-containers.sh

```diff
@@ -17,6 +17,7 @@ yarnAuditVersion=$(docker run --rm huskyci/yarnaudit:latest yarn audit --version
 safetyVersion=$(docker run --rm huskyci/safety:latest safety --version | awk -F " " '{print $3}')
 gitleaksVersion=$(docker run --rm huskyci/gitleaks:latest gitleaks --version)
 spotbugsVersion=$(docker run --rm huskyci/spotbugs:latest cat /opt/spotbugs/version)
+tfsecVersion=$(docker run --rm huskyci/tfsec:latest tfsec -v)
```

Comment:
Could you change to `./tfsec`, please? I was facing the following error:

```
$ docker run --rm huskyci/tfsec:latest tfsec -v
docker: Error response from daemon: OCI runtime create failed: container_linux.go:349: starting container process caused "exec: \"tfsec\": executable file not found in $PATH": unknown.
```

Reply:
Done :)